### DISCUSSION TITLE
Resources do not appear in the sidebar menu

### DISCUSSION THREAD

**Discussion by Denysetya1** [tomatophp/filament-translations#34] [Q&A] [answered]
Hi, why am I after completing the installation (adding it to the plugin in adminpanelprovider) the translation resource doesn't appear.
Now I get the error 
Route [filament.admin.resources.translations.index] not defined.

**Comment by arnaud-ritti** [comment, marked as answer]
I had the same issue. I've executed `php artisan optimize:clear` and it was fixed.

**Comment by fadymondy** [comment]
i know why this happened and i fix it 

https://github.com/tomatophp/filament-translations/releases/tag/v2.0.5
